embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/serialization/EmbraceSerializer.kt

```diff
@@ -44,6 +44,13 @@ internal class EmbraceSerializer {
         }
     }
 
+    fun <T> toJsonFromList(src: List<T>, clazz: Class<T>): String {
+        val adapter = impl.adapter<List<T>>(
+            Types.newParameterizedType(List::class.java, clazz)
```

Comment:
I'm fairly sure that you can use `fun <T> toJson(src: T, type: Type)` to achieve this without adding a new function. That should also be the case for the new `fromJson` overload that was added. Ping me if you need assistance as I spent a bit of time tackling this

Reply:
Updated. I had to add a new fromJson that receives the InputStream and the Type, as it was there for toJson but not for fromJson.